jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/logging/Verbosity.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.cli.cli2.logging;
+
+public enum Verbosity {
+  quiet(0),
+  error(1),
+  warn(2),
+  lifecycle(3),
+  info(4),
+  debug(5);
+  private final int value;
+
+  Verbosity(int value) {
+    this.value = value;
+  }
+
+  public int value() {
+    return value;
+  }
+
+  public boolean atLeast(Verbosity target) {
```

Comment:
Just to confirm, since the constructor is package-private: should these method  be `public`?

Reply:
So enum constructors cannot be public or protected. I think it's fine to expose these side-effect free methods as public, especially if we do verbosity checks/config elsewhere in the code?